.pylintrc

```diff
@@ -367,7 +367,7 @@ max-bool-expr=5
 max-branches=25
 
 # Maximum number of locals for function / method body
-max-locals=25
+max-locals=30
```

Comment:
True. Pylint was complaining that the `Alert` class had too many attributes. I felt it made sense as is, but if it grew considerably it should be refactored, which is why I raised this threshold instead of disabling the check on the class, for example. Do you feel we should restructure the class?